### PR TITLE
Update DevShell to write entities via the new storage stack

### PR DESCRIPTION
**Pull request by mykmartin**

### CODE REVIEW

File: shells/dev-shell/output-pane.js [hunk 1]
```diff
@@ -244,8 +240,7 @@ class ArcPanel extends HTMLElement {
       action = 'show';
     }
     for (const store of this.stores.children) {
-      // TODO: remove 'if' when saving entities works
```
Comment (Cypher1) [inline shells/dev-shell/output-pane.js:247]: Yay

File: shells/dev-shell/store-panel.js [hunk 2]
```diff
@@ -221,13 +219,24 @@ export class StorePanel extends HTMLElement {
...
+          // A new entity has been added.
+          entity = this.makeEntity(id, data);
+        }
+        if (entity) {
+          await this.handle.add(entity, [String(Math.random()).slice(2)]);
         }
       }
       for (const id of curIds.values()) {
-        await this.handle.remove(id);
+        await this.handle.remove(this.makeEntity(id, {}));
```
Comment (Cypher1) [inline shells/dev-shell/store-panel.js:239]: I really like this. Neat

**Comment by Cypher1** [comment]
Running the cloud build again to see if the timeouts are flakey

**Comment by mykmartin** [comment]
> Running the cloud build again to see if the timeouts are flakey

Thanks; they are. There is no way this PR could be causing any tests to fail...

**Comment by Cypher1** [comment]
Hmm, I probably wasted a bit of cloud time then. Ah well.
Thanks Myk

Joshua Pratt | Arcs | jopra@google.com

**Pull request merged**